appserver/admingui/microprofile-console-plugin/src/main/resources/fish/payara/admingui/microprofile/Strings.properties

```diff
@@ -69,13 +69,17 @@ faultTolerance.configuration.managedScheduledExecutorServiceNameHelp=The JNDI na
 
 healthCheck.configuration.enabled=Enabled
 healthCheck.configuration.enabledHelp=Enables or Disables the Health Check.
+healthCheck.configuration.securityEnabled=Enable Secure Access
```

Comment:
Change the text to "Require authorization"

Reply:
"Enable Secure Access" label is added here to maintain consistency across other endpoints like REST Monitoring.

Reply:
The wording in REST Monitoring is can be wrong so I wouldn't blindly follow it for keeping consistency. Both should rather say "Require Secure Access" or "Require secure authorization". But I'm not going to argue, we can leave it as it is.

Reply:
The wording in REST Monitoring is can be wrong so I wouldn't blindly follow it for keeping consistency. Both should rather say "Require Secure Access" or "Require secure authorization". But I'm not going to argue, we can leave it as it is.